servers/quarkus-auth/src/test/java/org/projectnessie/server/authz/TestCELAuthZ.java

```diff
@@ -52,10 +52,11 @@ public class TestCELAuthZ {
   public void addsViewAllRefsRule() throws ScriptException {
     CompiledAuthorizationRules rules = new CompiledAuthorizationRules(buildConfig(true));
     soft.assertThat(rules.getRules())
-        .hasSize(4)
+        .hasSize(5)
```

Comment:
Maybe change to `.keySet()).containsExactlyInAnyOrder(......)`?

Reply:
done